user-interface/src/main/java/life/qbic/datamanager/views/demo/ComponentDemo.java

```diff
@@ -220,6 +220,39 @@ public UserInput userInput() {
     return steps;
   }
 
+  private static Div dialogWithOneButton(AppDialog dialog, String dialogType) {
+    Div content = new Div();
+    Button showDialog = new Button("Show Dialog");
+    // Dialog set-up
+    DialogHeader.withIcon(dialog, dialogType, IconFactory.warningIcon());
+    DialogFooter.withConfirmOnly(dialog, "Close");
+    ExampleUserInput userInput = new ExampleUserInput("Expelliarmus");
+    DialogBody.with(dialog, userInput, userInput);
+
+    Div confirmBox = new Div("Click the button and press 'Cancel' or 'Save'");
+    showDialog.addClickListener(e -> {
+      dialog.open();
+      confirmBox.setText("Cancelled the dialog.");
+    });
+
+    dialog.registerCancelAction(() -> {
+      dialog.close();
+      if (dialog.hasChanges()) {
+        confirmBox.setText("Cancelled the dialog although there where changes made!");
+      } else {
+        confirmBox.setText("Cancelled the dialog. No changes.");
+      }
+    });
```

Comment:
Is this even possible when the footer only has a confirm button?

Reply:
you can set it, but it cant be executed :)